src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1044,8 +1044,6 @@ private function findNamespacePrefixes($xpath)
      */
     private function createSubCrawler($nodes)
     {
-        $crawler = new static($nodes, $this->uri, $this->baseHref);
-
-        return $crawler;
+        return new static($nodes, $this->uri, $this->baseHref);
```

Comment:
should be reverted to simplify merging between branches (as of 2.8, we have additional code before the return statement)